filament/src/PostProcessManager.cpp

```diff
@@ -316,6 +320,148 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::fxaa(FrameGraph& fg,
     return ppFXAA.getData().output;
 }
 
+FrameGraphId<FrameGraphTexture> PostProcessManager::dof(FrameGraph& fg,
+        FrameGraphId<FrameGraphTexture> input,
+        const View::DepthOfFieldOptions& dofOptions,
+        const CameraInfo& cameraInfo) noexcept {
+
+    FEngine& engine = mEngine;
+    Handle<HwRenderPrimitive> const& fullScreenRenderPrimitive = engine.getFullScreenRenderPrimitive();
+
+    FrameGraphId<FrameGraphTexture> depth = fg.getBlackboard().get<FrameGraphTexture>("structure");
+    assert(depth.isValid());
+
+    const size_t sampleCount = 25;  // (keep in sync with dofUtils.fs)
```

Comment:
Should we control this with a quality setting?

Reply:
Maybe we should just make it a value the user can set, after all.